cmdeploy/src/cmdeploy/__init__.py

```diff
@@ -375,17 +387,26 @@ def _configure_dovecot(config: Config, debug: bool = False) -> bool:
     return need_restart
 
 
-def _configure_nginx(domain: str, debug: bool = False) -> bool:
+def _configure_nginx(config: Config, domain: str, debug: bool = False) -> bool:
```

Comment:
Hm, `domain` == `config.mail_domain`, so I think it isn't very pretty to pass it to this method, instead it should be used from the config object in the whole method. Can you change this?

Reply:
done